string command
string arg1
string arg2
